io.openems.edge.evcs.api/src/io/openems/edge/evcs/api/Evcs.java

```diff
@@ -70,25 +80,81 @@ public enum ChannelId implements io.openems.edge.common.channel.ChannelId {
 		 *
 		 * <p>
 		 * This value is derived from the charging station or calculated during the
-		 * charging.
-		 *
+		 * charging. When this value is set, the minimum and maximum limits are set at
+		 * the same time if the EVCS is a {@link ManagedEvcs}.
+		 * 
 		 * <ul>
 		 * <li>Interface: Evcs
 		 * <li>Readable
 		 * <li>Type: Integer
 		 * </ul>
 		 */
-		PHASES(Doc.of(OpenemsType.INTEGER) //
+		PHASES(Doc.of(Phases.values()) //
+				.debounce(5) //
```

Comment:
Is there any reason to add the `decounce(5)`?

Reply:
On some charging station models the phases are not switched consistently. Instead within the very first seconds they are switch several times, e.g. from 3 to 1 to 3 to 1. I have seen that often.

Reply:
Thank you!

For EVCS with fixed phases, usually it's simply set during initialization:
https://github.com/OpenEMS/openems/blob/c78334d8810d2a291511a9fa3e1c6072c03da84a/io.openems.edge.evcs.hardybarth/src/io/openems/edge/evcs/hardybarth/EvcsHardyBarthImpl.java#L79
which "works" for 3 phase EVCS, since by default this channel is 3 phases.

However, we found that for our case of only 1 phase, the `this._setPhases(Phases.ONE_PHASE);` does not work.
Does it mean we need to repeat the `this._setPhases(Phases.ONE_PHASE);` for 5 times?